jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/LDAPUserInfoImpl.java

```diff
@@ -178,7 +178,7 @@ public String getEntityForEmail(String email) {
         String context = getConfigProperty(USER_CTX);
         String filter = getConfigProperty(EMAIL_FILTER);
         String attributeId = getConfigProperty(USER_ATTR_ID, DEFAULT_USER_ATTR_ID);
-        LdapSearcher searcher = ldapSearcher.search(context, filter, email);
+        final LdapResults searcher = ldapSearcher.search(context, filter, email);
```

Comment:
sorry, I can't see why we need to declare it as `final`?

Reply:
it does not any value a makes complex debugging.

Reply:
ok, removed final